MIP101/MIP101.md

```diff
@@ -395,186 +395,9 @@ In severe cases that can be described as Governance Attacks, in addition to bein
 
 Alignment Conservers are encouraged to be anonymous. Some Alignment Conserver roles are required to be anonymous, and their identities must be derecognized from acting as Alignment Conservers in general, in case their privacy is breached.
```

Comment:
Having only two Alignment Conserver roles left, can we make this more specific? ADs must be anonymous. What about Facilitators? As far as I know, currently only Governance Facilitators are anonymous. This should also be aligned with section 2.7.5 (Facilitators).